openwisp_radius/views.py

```diff
@@ -0,0 +1,27 @@
+from private_storage.views import PrivateStorageDetailView
+from swapper import load_model
+
+from .base.models import _get_csv_file_private_storage
+
+OrganizationUser = load_model('openwisp_users', 'OrganizationUser')
+
+
+class RadiusBatchCsvDownloadView(PrivateStorageDetailView):
+    storage = _get_csv_file_private_storage
+    model = load_model('openwisp_radius', 'RadiusBatch')
+    model_file_field = 'csvfile'
+    slug_field = 'csvfile'
+    slug_url_kwarg = 'csvfile'
+
+    def can_access_file(self, private_file):
+        user = private_file.request.user
+        if not user.is_authenticated:
+            return False
+        org = (self.object.organization.pk,)
```

Comment:
we should now simplify this block of code with just the following:

```python
return user.is_superuser or (user.is_staff and user.is_manager(self.object.organization))
```

See [Model helpers in openwisp-users](https://github.com/openwisp/openwisp-users/#model-helpers).

Reply:
Done! :smile: